test/grep.js

```diff
@@ -49,6 +49,13 @@ test('if at least one file is missing, this should be an error', t => {
   t.is(result.code, 2);
 });
 
+test("multiple files, one doesn't exist, one doesn't match", t => {
+  const result = shell.grep(/oogabooga/, 'test/resources/file1.txt',
+    'test/resources/thisonedne');
```

Comment:
Generally prefer to expand abbreviations:

```suggestion
    'test/resources/filedoesnotexist.txt');
```